gp-okta.conf

```diff
@@ -2,15 +2,15 @@
 debug = 0
 vpn_url = https://grandrounds.gpcloudservice.com 
 okta_url = https://grandrounds.okta.com
-username = marcelo.vanzin
+username = rick.cobb
 # password = mypass
 # mfa_order = totp sms
 # sms.okta = 0
 # totp.okta = ABCDEFGHIJKLMNOP
 # totp.google = ABCDEFGHIJKLMNOP
 gateway = US East
 openconnect_cmd = sudo openconnect
-openconnect_args =                    # optional arguments to openconnect
+openconnect_args = --csd-wrapper /usr/lib/x86_64-linux-gnu/openconnect/hipreport.sh
```

Comment:
i'd prefer this to be set via environment variable and default to a vendored hipreport.sh

e.g. `GP_OPENCONNECT_ARGS="--csd-wrapper $GR_HOME/pan-globalprotect-okta/hipreport.sh"`